### PR TITLE
Update for gazebo8 in gazebo_underwater

### PR DESCRIPTION
**Pull request by joaobrittoneto**
Depends on
- [ ] https://github.com/rock-gazebo/simulation-gazebo_underwater/pull/1

### CODE REVIEW

File: src/GazeboThruster.cpp [hunk 1]
```diff
@@ -191,23 +192,23 @@ void GazeboThruster::updateBegin(common::UpdateInfo const& info)
...
-        gazebo_underwater::Vector6 effort( force, (link->GetRelativePose().pos-cog).Cross(force));
+        Vector3d force(GzGetIgn((*link), RelativePose, ()).Rot().RotateVector(Vector3d::UnitX));
+        gazebo_underwater::Vector6 effort( force, (GzGetIgn((*link), RelativePose, ()).Pos()-cog).Cross(force));
         effort = matrix * effort;
-        thruster->added_mass_compensated_direction = link->GetRelativePose().rot.RotateVectorReverse(effort.top);
-        thruster->added_mass_compensated_position = effort.top.Cross(effort.bottom)/(thruster->added_mass_compensated_direction.GetSquaredLength()) + cog - link->GetRelativePose().pos;
-        thruster->added_mass_compensated_position = link->GetRelativePose().rot.RotateVectorReverse(thruster->added_mass_compensated_position);
+        thruster->added_mass_compensated_direction = GzGetIgn((*link), RelativePose, ()).Rot().RotateVectorReverse(effort.top);
+        thruster->added_mass_compensated_position = effort.top.Cross(effort.bottom)/(gazebo::math::Vector3(thruster->added_mass_compensated_direction).GetSquaredLength()) + cog - GzGetIgn((*link), RelativePose, ()).Pos();
```
Comment (joaobrittoneto) [inline src/GazeboThruster.cpp:211]: @doudou 
I could not find an equivalent of `GetSquaredLength` for ignition

**Comment by doudou** [comment]
Hey ... wrong target. Since I forked the repo, the default PR target is BIR -- I don't know how to change that.

**Comment by joaobrittoneto** [comment]
I added the rock_gazebo as the remote repository and did the push there.
I don't know why it came to BIR.

**Comment by joaobrittoneto** [comment]
Ah ok, sorry....
I open the PR in [rock-gazebo](https://github.com/rock-gazebo/simulation-gazebo_thruster/pull/1).
Closing this one.

**Pull request closed without merging**